<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InitDialog</class>
 <widget class="QDialog" name="InitDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>299</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget_4" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>461</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QGroupBox" name="groupBoxPlayer">
      <property name="title">
       <string>Select Player:</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="radioButtonA">
         <property name="text">
          <string>Player A</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButtonB">
         <property name="text">
          <string>Player B</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButtonC">
         <property name="text">
          <string>Player C</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBoxAddress">
      <property name="title">
       <string>IP Addresses and Ports:</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QWidget" name="widget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="labelA">
            <property name="text">
             <string>Player A</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditA"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_2" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="labelB">
            <property name="text">
             <string>Player B</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditB"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_3" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="labelC">
            <property name="text">
             <string>Player C</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditC"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>260</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>I'm Ready!</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
